### DISCUSSION TITLE
Multiple instances with the same Telegram account

### DISCUSSION THREAD

**Discussion by Digi777** [aahnik/tgcf#379] [General]
Hi, is there a risk to use 2 or 3 instances of TGCF and link them to 1 Telegram account in user mode? The IP addreses of the instances in my case would be in the same geographic area.

**Comment by navtemmt** [comment]
I tried running multiple instance of tgcf in same web browser with different port and experiencing the forward stopped after a while and you need restart it to make it work

**Comment by Digi777** [comment]
Yeah I finally chose to buy a new sim card and deploy a 2nd instance. Way more stable solution aha
